code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -190,7 +190,6 @@
 		else
 			to_chat(user, span_warning("The queen bee disappeared! Disappearing bees have been in the news lately..."))
 
-		qdel(qb)
```

Comment:
where'd this go?

Reply:
the `forceMove()` above calls `Exited()` and deletes it, calling `qdel()` again qdels a qdeleted object

Reply:
aight